addon/components/frost-multi-select.js

```diff
@@ -54,7 +54,28 @@ let FrostMultiSelect = FrostSelect.extend({
       this._super(term)
     }
   },
+  selectOptionByValue (selectedValue) {
+    if (_.isUndefined(selectedValue)) {
+      return
+    }
+
+    let items = this.get('items')
 
+    if (!_.isArray(selectedValue)) {
+      selectedValue = [selectedValue]
+    }
+    let selected = selectedValue.map(function (value) {
+      return _.findIndex(items, (item) => _.isEqual(item.value, value))
+    }).filter(function (value) {
+      return value >= 0
```

Comment:
So you are ensuring negative numbers are not in the `selectedValue` array because the array is a list of indexes?